README.md

```diff
@@ -301,11 +363,14 @@ CSS animations too.
 ```js
 const MyComponent = () => {
   const [focusedSection, setFocus] = useState('foo')
+  const isFocused = name => x => name === x ? 'foreground' : 'background'
```

Comment:
Was this example not working before? What is supposed to be happening in this example now? It looks like the `focusedSection` variable isn't being used anymore, so I think this code isn't doing what you thought unless I'm missing something.

Reply:
Yikes, thanks for the sharp eyes. I should've pulled it into a sandbox to test it in full first. Fixed now.